.travis.yml

```diff
@@ -77,8 +77,7 @@ matrix:
       env: REPO=repo-skins BRANCH=leia
 
 # command to install dependencies
-install:
-  - pip install -r requirements.txt
+install: "make"
```

Comment:
Has to be squashed into the commit where `requirements.txt` is removed or committed before that.
Any reason to change style?
```
install:
  - make
```

Reply:
Okay then I'll have to squash the last three commits.

No reason for style change. Will make it the old way.